### PR TITLE
Clean setup by using jupyter-packaging

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
@jtpio would you mind reviewing this?

**Comment by jtpio** [comment]
Thanks, looks good at first glance.

Maybe there could be a step on CI that checks the `nbextension` is properly installed?

Similar to: https://github.com/QuantStack/ipytree/blob/f6ee37f9a8123f14eae5485e22fe5e4b5d9e8e9b/.github/workflows/build.yml#L31

File: .travis.yml [hunk 1]
```diff
@@ -31,15 +31,16 @@ install:
...
-    - if [[ "$TRAVIS_PYTHON_VERSION" == "3.6" ]]; then
-        export DISPLAY=:99.0;
-        sh -e /etc/init.d/xvfb start || true;
-        jupyter labextension install js @jupyter-widgets/jupyterlab-manager;
-        python -m jupyterlab.selenium_check;
-        (rm -rf js/node_modules);
-        (cd js; npm install);
-        (cd js; npm run test);
-      fi
+  - jupyter nbextension list 2>&1 | grep -ie "ipytree/extension.*enabled" -
```
Comment (jtpio) [inline .travis.yml:34]:
```suggestion
  - jupyter nbextension list 2>&1 | grep -ie "bqplot/extension.*enabled" -
```

Comment (martinRenou) [reply .travis.yml]: Thanks!

**Comment by martinRenou** [comment]
@jtpio ok to merge?

**Comment by jtpio** [comment]
Looks like a nice cleanup, thanks @martinRenou :+1: 

One thing to look at is the content of the sdist, to make sure all the required files are there. If that's the case then it should be good to go :rocket:

**Comment by martinRenou** [comment]
It looks good to me, they both contain the same files except `pyproject.toml`, which is expected.

**Pull request merged**